docs/docs/general/02-Getting Started/writing-your-first-test.md

```diff
@@ -19,7 +19,25 @@ This is going to create
   - `test/my-first-askui-test-suite.test.ts`: an example test with askui,
   - a folder `helper` which contains `jest.setup.ts` file: this file is setting up the test environment for all tests
   - a `jest.config.ts`: file for configuration of Jest,
-  
+
+### Configuration
+
+We need you to create some credentials through our [askui user portal](https://app.askui.com/) (usage is free!).
+Please see <a href="../askui%20User%20Portal/signup" target="_blank">our documentation on how to signup and create the credentials</a>.
```

Comment:
I don't like using html in markdown when markdown can still be used because of tooling, complexity etc. Do we need it here or can't we simply use the classical markdown way of describing external links (https://anvilproject.org/guides/content/creating-links)?

Reply:
I do not like it either :( 

Problem is that this is an internal link and per default Docusaurus opens internal links in the same tab without much to do about it.
This is due to the relative link processing from the rendering plugin.

External links are opened in a new tab.

I will do some digging here. Maybe it is possible...

Reply:
Sounds like too much effort for now. I would keep the HTML. But in the end it is your decision :)

Reply:
Let us keep it for now. But I will add it to our backlog. Feels like the plugin system is very powerful and worth to learn!